wp-content/themes/borderzine/inc/widgets/class-borderzine-6-col-widget.php

```diff
@@ -0,0 +1,301 @@
+		}
+		if ( ! empty( $instance['tag'] ) ) {
+			$query_args['tag'] = $instance['tag'];
+		}
+		if ( ! empty( $instance['author'] ) ) {
+			$query_args['author'] = $instance['author'];
+		}
+		if ( ! empty( $instance['taxonomy'] ) && ! empty( $instance['term'] ) ) {
+			$query_args['tax_query'] = array(
+				array(
+					'taxonomy'	=> $instance['taxonomy'],
+					'field' 	=> 'slug',
+					'terms' 	=> $instance['term']
+				)
+			);
+		}
+
+		echo '<ul>';
+
+		$my_query = new WP_Query( $query_args );
+
+		if ( $my_query->have_posts() ) {
+
+			$output = '';
+
+			while ( $my_query->have_posts() ) : $my_query->the_post(); $shown_ids[] = get_the_ID();
+
+				// wrap the items in li's.
+				$output .= '<li>';
+
+				$context = array(
+					'instance' => $instance,
+					'thumb' => $thumb,
+					'excerpt' => $excerpt
+				);
+
+				ob_start();
+				largo_render_template( 'partials/widget', 'content', $context );
+				$output .= ob_get_clean();
+
+				// close the item
+				$output .= '</li>';
+
+			endwhile;
+
+			// print all of the items
+			echo $output;
+
+		} else {
+			printf( __( '<p class="error"><strong>You don\'t have any recent %s.</strong></p>', 'largo' ), strtolower( $posts_term ) );
+		} // end more featured posts
+
+		// close the ul
+		echo '</ul>';
+
+		echo $args['after_widget'];
+
+		// Restore global $post
+		wp_reset_postdata();
+		$post = $preserve;
+	}
+
+	function update( $new_instance, $old_instance ) {
+		$instance = $old_instance;
+		$instance['title'] = sanitize_text_field( $new_instance['title'] );
+		$instance['num_posts'] = intval( $new_instance['num_posts'] );
+		$instance['avoid_duplicates'] = ! empty( $new_instance['avoid_duplicates'] ) ? 1 : 0;
+		$instance['thumbnail_display'] = sanitize_key( $new_instance['thumbnail_display'] );
+		$instance['image_align'] = sanitize_key( $new_instance['image_align'] );
+		$instance['excerpt_display'] = sanitize_key( $new_instance['excerpt_display'] );
+		$instance['num_sentences'] = intval( $new_instance['num_sentences'] );
+		$instance['show_byline'] = ! empty($new_instance['show_byline']);
+		$instance['hide_byline_date'] = ! empty($new_instance['hide_byline_date']);
+		$instance['show_top_term'] = ! empty($new_instance['show_top_term']);
+		$instance['show_icon'] = ! empty($new_instance['show_icon']);
+		$instance['cat'] = intval( $new_instance['cat'] );
+		$instance['tag'] = sanitize_text_field( $new_instance['tag'] );
+		$instance['taxonomy'] = sanitize_text_field( $new_instance['taxonomy'] );
+		$instance['term'] = sanitize_text_field( $new_instance['term'] );
+		$instance['author'] = intval( $new_instance['author'] );
+		$instance['linktext'] = sanitize_text_field( $new_instance['linktext'] );
+		$instance['linkurl'] = esc_url_raw( $new_instance['linkurl'] );
+		return $instance;
+	}
+
+	function form( $instance ) {
+		$defaults = array(
+			'title' => __( 'Recent ' . of_get_option( 'posts_term_plural', 'Posts' ), 'largo' ),
+			'num_posts' => 5,
```

Comment:
default should be 6

Reply:
@benlk Resolved in https://github.com/INN/umbrella-borderzine/pull/75/commits/f766833d9593870549f624102f4c410abd663258.